### PR TITLE
global: port to new structure

### PR DESCRIPTION
**Pull request by jirikuncar**
- [x] amend tests
- [x] depends on https://github.com/inveniosoftware/invenio-oauth2server/pull/22

### CODE REVIEW

File: invenio_webhooks/templates/invenio_webhooks/base.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```
Comment (lnielsen) [inline invenio_webhooks/templates/invenio_webhooks/base.html]: I think this one can be removed.

File: invenio_webhooks/templates/invenio_webhooks/index.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{%- extends config.WEBHOOKS_BASE_TEMPLATE %}
```
Comment (lnielsen) [inline invenio_webhooks/templates/invenio_webhooks/index.html]: this one as well.

**Comment by jirikuncar** [comment]
@lnielsen ping https://travis-ci.org/jirikuncar/invenio-webhooks/builds/95543870

**Comment by jirikuncar** [comment]
@lnielsen travis is green :green_heart:

**Pull request merged**